packages/styletron-react/src/index.js

```diff
@@ -30,16 +30,23 @@ import {addDebugMetadata, DebugEngine} from "./dev-tool.js";
 
 export {DebugEngine};
 
-const StyletronContext: Context<any> = createReactContext();
-const HydrationContext = createReactContext(false);
-const DebugEngineContext = createReactContext();
+const StyletronContext = React.createContext<StandardEngine>({
+  renderStyle: () => "",
+  renderKeyframes: () => "",
+  renderFontFace: () => "",
+  noopEngine: true,
```

Comment:
I think rather than duck typing via an extra `noopEngine` property, it may be more clear to assign this object to a variable and then check via reference equality in `checkNoopEngine`.

At first I thought `noopEngine` might have something to do with enabling/disabling this feature (but that'd be a bit nonsensical). Also since this is basically a "private" property, it'd be nice to not have it show up in the `StandardEngine` type.

Reply:
Good point! Updated.